### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by HAL9081** [nefarius/DsHidMini#163] [Show and tell]
Hello team,

thank you for ViGEm.
It works like a charm.

I'm currently using it to connect my original DS3 controller via Bluetooth.

I'm looking forward to the integration of features like found in SCP Toolkit.
(Flip Axis, Dead Zone Threshold, LED Status choices)

Thanks for your work and dedication.

Best regards
HAL

**Comment by nefarius** [comment]
Thank you for the encouraging words, it's highly appreciated! ❤️
